bundles/io.openliberty.tools.eclipse.ui/META-INF/MANIFEST.MF

```diff
@@ -29,6 +29,7 @@ Import-Package: org.eclipse.core.commands,
  org.eclipse.jdt.launching,
  org.eclipse.jem.util.emf.workbench,
  org.eclipse.osgi.service.debug,
+ org.eclipse.osgi.util;version="1.1.0",
```

Comment:
Seems like we'd want to avoid tying to a specific version unless there's a special need.

Reply:
According to this stackoverflow answer, it doesnt look like we'd be able to lookup messages outside the current locale.
https://stackoverflow.com/questions/21334606/using-eclipse-externalized-strings-how-to-get-a-string-in-language-xyz

I think moving forward we'd be less aligned with the messages printed in trace and the user facing messages (tracing being less formal or shorter, like you said).